app/mailers/feedback_messages/feedback_base.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module FeedbackMessages
+  class FeedbackBase
+    def initialize(claim, comment = '')
+      @claim = claim
+      @comment = comment
+    end
+
+    def template
+      raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
+    end
+
+    def contents
+      raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
+    end
+
+    def recipient
+      raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
+    end
+
+    protected
+
+    def main_defendant
+      @claim.data['defendants'].find { |defendant| defendant['main'] == true }
```

Comment:
You _could_ shorten this to:
```suggestion
      @claim.data['defendants'].find { _1['main'] }
```
(The `== true` is redundant unless `main` contains non-boolean values like strings that should be evaluated as `false`, and given that each item in `defendants` is a defendant by implication, the `_1` shorthand instead of a named variable makes things more concise (although depending on how used you are to it, you may find it harder to read).

Reply:
only a personal opinion and I'm happy to be overruled but my preference would be to not use the _1 shorthand to make the code more explicit